### ISSUE TITLE
[Bug]: **Many** Items cannot be Dragged and dropped onto a Form in Designer

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#92]
**Describe the bug**
Whilst testing #91 reproduction steps in Standard-Toolkit-Demo's I found that many of the toolkit items cannot simple be dragged and dropped onto a from.

Also of those items that have successfully been draged, then they cannot be deleted ! (This may be due to cascade error form the initial designer failures)

**To Reproduce**
![SomeDoNotWork](https://user-images.githubusercontent.com/2418812/117565278-43989700-b0a8-11eb-80dc-871974039ed0.gif)

**Expected behaviour**
Be able to use designer for **ALL* Krypton elements

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
```
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
  <PropertyGroup>
    <TargetFrameworks>net48;net5.0-windows</TargetFrameworks>
    <OutputType>WinExe</OutputType>
    <RootNamespace>Test_Clip_Base</RootNamespace>
    <AssemblyName>Test Clip Base</AssemblyName>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
    <UseWindowsForms>true</UseWindowsForms>
    <PlatformTarget>AnyCPU</PlatformTarget>
  </PropertyGroup>
```

**Additional context**
`<ProjectReference Include="..\..\..\..\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Krypton.Toolkit 2019.csproj" />`

**Comment by Wagnerp** [comment]
I had the exact same thing happening in a pure .NET project, I don't know if this is a compatibility issue?

**Comment by Smurf-IV** [comment]
I can confirm that the delete error is seen once the an initial "Drag-Drop" error is seen :-(

**Comment by Wagnerp** [comment]
Are bugs #54, #66 and #71 all interlinked with this one?

**Comment by Smurf-IV** [comment]
Not sure #66 is as that is a different error (Root cause might be the same tho'...)
I have taken 54 and 71 as part of this investigation..

**Comment by Smurf-IV** [comment]
Drag'n'Dropped them all 👍 
![image](https://user-images.githubusercontent.com/2418812/118356801-e3559980-b56e-11eb-8ce1-be2c3a3517c0.png)

**Comment by Smurf-IV** [comment]
As shown above.. All is "_Good_" now when the Form is just a `Form`.
**BUT**
Change it to a `KryptonForm`, and it all goes very wrong !!!!!

**Comment by Smurf-IV** [comment]
The following needs to go into a FAQ - As it is a _Feature_ of Visual Studio (16.9.x anyway!)

With the fixes I have done.. 

It is possible to load _all_ of the Toolkits into a Target application that has a **Single* Target framework.

**IF** the project is MultiTarget: Then it is **NOT** possible to use _some_ of the Toolkit designer elements !

At least when Linking directly from the source of the Toolkit to the Project (Same _will_ probably apply via NuGet as the Designer will not know which to use first !)

**Comment by Smurf-IV** [comment]
Yay.. Found the workaround.. But it has to go into the Forms project....

- Close any designer forms you may have open
- Place the following into the csproj file:
``` c#
    <ItemGroup>
        <Reference Include="System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net4'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>4.0.0.0</Version>
            <TreatAsUsed>False</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net5'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>5.0.0.0</Version>
            <TreatAsUsed>False</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net6'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>6.0.0.0</Version>
            <TreatAsUsed>False</TreatAsUsed>
        </Reference>
        <Reference Include="System.Design" Condition="'$(TargetFramework)' == 'net35'" />
        <!--TODO: Need to specify version here-->
    </ItemGroup>

```
- Rebuild.
  - You _May_ have to close Visual Studio and reopen / reload
- Open the Form and you will have the designers available to use :-)

**Comment by Smurf-IV** [comment]
Available August 2021 (Canary)

**Comment by Smurf-IV** [comment]
![image](https://user-images.githubusercontent.com/2418812/127779675-b9fd9190-9178-43d8-82e7-c05c6353ebdc.png)

**Comment by Smurf-IV** [comment]
It's the controls that have a designer that is derived from `ParentControlDesigner`

**Comment by Smurf-IV** [comment]
`ParentControlDesigner Class` has been moved into `System.Windows.Forms.Design.dll` since .netCore

**Comment by Wagnerp** [comment]
Available in build 2108.1
